src/connect4/core.rs

```diff
@@ -47,9 +47,9 @@ pub enum MyColor {
 
 /// Struct determines position on the board
 #[derive(Clone, Copy, PartialEq, Eq, Debug)]
-struct GridPosition {
-    x: i32,
-    y: i32,
+pub struct GridPosition {
+    pub x: i32,
+    pub y: i32,
```

Comment:
I don't know if this is ideal syntax...I like being able to write `grid.x`, for example, but feels weird to have `pub` fields in side the struct. If you want to use getter/setting like `getX()` or `setX()`, let me know. Probably a semi-large rework tho.

Reply:
For convenience, using pub instead of adding getter is alright...